### DISCUSSION TITLE
How can I fix the 'This Edge Function has crashed' error in Next.js 13.4.2 due to middleware.ts file?

### DISCUSSION THREAD

**Discussion by AmeizT** [vercel/community#2547] [Help]

**Comment by z4nr34l** [comment]
Could you bring us more details?
I consider as helpful those informations:
1. Repository Access;
2. Vercel build logs (from deployment with this issue);
3. Runtime logs from problematic request;

The more - the better for receiving quick help.

You can also try to reproduce this problem on free [CodeSandbox](https://codesandbox.io/) and post URL here when it's done.
